src/dsymbol/conversion/first.d

```diff
@@ -981,6 +977,9 @@ private:
 					addTypeToLookups(parameter.typeLookups, p.type);
 				parameter.parent = currentSymbol;
 				currentSymbol.acSymbol.argNames.insert(parameter.acSymbol.name);
+
+				currentSymbol.acSymbol.functionArguments ~= parameter.acSymbol;
```

Comment:
If you are saving parameters why is the array called functionArguments?

```
void foo(int x) // x is a parameter
foo(3); // 3 is an argument
```

Reply:
good point I will change it